src/acl/acl.test.ts

```diff
@@ -476,7 +476,7 @@ describe("createAcl", () => {
 });
 
 describe("createAclFromFallbackAcl", () => {
-  it("creates a new ACL including existing default rules as Resource rules", () => {
+  function expectCreatesNewAclWithDefault(defaultPredicate: string) {
```

Comment:
I'd prefer to keep tests self-contained, i.e. copy-paste the existing test but using `defaultForNew` instead of `default`, rather than extracting it into a function. (I think the acronym people like here is to "keep your tests DAMP".)

Reply:
yeaaa - that's what I was thinking too, thanks!